cli/bsc.js

```diff
@@ -5,8 +5,12 @@
 import { execFileSync } from "node:child_process";
 
 import { bsc_exe } from "./common/bins.js";
+import { runtimePath } from "./common/runtime.js";
 
 const delegate_args = process.argv.slice(2);
+if (!delegate_args.includes("-runtime-path")) {
```

Comment:
So this happens when we run `bunx bsc` directly. It makes me wonder if we could just reuse this and pass it to Rewatch via an environment variable.

I'm not against `oxc_resolver` per se, but doesn't this create two different mechanisms for resolving the `-runtime-path`?